src/api/utility-types.md

```diff
@@ -1,14 +1,14 @@
-# Utility Types {#utility-types}
+# Tipos de Utilitário de TypeScript {#utility-types}
 
 :::info
-This page only lists a few commonly used utility types that may need explanation for their usage. For a full list of exported types, consult the [source code](https://github.com/vuejs/core/blob/main/packages/runtime-core/src/index.ts#L131).
+Esta página lista apenas alguns tipos de utilitários comumente usados ​​que podem precisar de explicação de uso. Para obter uma lista completa dos tipos exportados, consulte o [código fonte](https://github.com/vuejs/core/blob/main/packages/runtime-core/src/index.ts#L131).
 :::
 
 ## PropType\<T> {#proptype-t}
 
-Used to annotate a prop with more advanced types when using runtime props declarations.
+Usado para anotar uma propriedade com tipos mais avançados ao usar declarações de pro em tempo de execução.
```

Comment:
There's probably a typo in this line, I think it should be "declarações de propriedades" instead